app/code/Magento/Email/Test/Unit/Model/Plugin/WindowsSmtpConfigTest.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Email\Test\Unit\Model\Plugin;
+
+use Magento\Email\Model\Plugin\WindowsSmtpConfig;
+use Magento\Framework\App\Config\ReinitableConfigInterface;
+use Magento\Framework\Mail\TransportInterface;
+use Magento\Framework\OsInfo;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+
+/**
+ *  WindowsSmtpConfigTest
+ */
+class WindowsSmtpConfigTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * @var WindowsSmtpConfig|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $windowsSmtpConfigMock;
+
+    /**
+     * @var OsInfo|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $osInfoMock;
+
+    /**
+     * @var ReinitableConfigInterface|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $configMock;
+
+    /**
+     * @var TransportInterface
+     */
+    private $transportMock;
+
+    /**
+     * setUp
+     *
+     * @return void
+     */
+    public function setUp()
+    {
+        $objectManager = new ObjectManager($this);
+
+        $this->osInfoMock = $this->createMock(OsInfo::class);
+        $this->configMock = $this->createMock(ReinitableConfigInterface::class);
+        $this->transportMock = $this->createMock(TransportInterface::class);
+
+        $this->windowsSmtpConfigMock = $objectManager->getObject(
```

Comment:
It does not look like a "mock". We test the real object. Please, fix the naming.